### ISSUE TITLE
`get_vessel_info()` : `search_type` should have default value

### ISSUE THREAD

**Issue by teuder** [GlobalFishingWatch/gfwr#30]
Currently, when the user does not specify `search_type` it shows an error message, which is unclear to the user.

```
Error in if (search_type == "id") { : argument is of length zero
```

I think `search_type = "basic"` would be good default value.

And we should output an error message like when the user specified no value or wrong value to the argument `search_type`.

```r
if(!search_type %in% c("basic", "advanced", "id")){
	stop("Please specify 'basic', 'advanced' or 'id' for the argument `search_type`.")
```

**Comment by tclavelle** [comment]
@natemiller Would this issue be closed by #74 ?

**Comment by natemiller** [comment]
Addressed by #75
